packages/tupaia-web/src/utils/useDateRanges.ts

```diff
@@ -112,7 +112,6 @@ export const useDateRanges = (
 
   const { startDate: urlStartDate, endDate: urlEndDate } = convertUrlPeriodStringToDateRange(
     currentPeriodString,
-    periodGranularity,
```

Comment:
Why do we no longer need periodGranularity? Won't it always now default to day?

Reply:
We do use the period granularity for the display, but when converting to and from the URL search parameters the current way is to use an actual full date range (e.g. 01_April_2023-30_April_2023) as opposed to the display equivalent (i.e.g Apr_2023) because the query won't work with the display version